server/lib/environment.js

```diff
@@ -1,6 +1,9 @@
 const davEnv = process.env.DAV_ENV;
 
-const hasStore = ['simulated'].includes(davEnv);
+function hasStore() {
```

Comment:
I feel like a function expression would work more predictably than a function declaration... especially if we make any changes to the function in the future (which we will to support other environments).

You can read more about the difference here: https://kangax.github.io/nfe/ (I had to read it to remember the exact difference and nomenclature.)

In addition, with a function expression we can do an arrow functions, and those are simply 🔥 hot.

I'm thinking something like:
```javascript
const hasStore = () => ['simulated'].includes(davEnv);
```

Reply:
Wow. I had to read that article three times with a ten minute break in between each time for it to really sink in. Awesome bookmark for future reference though, thanks for sharing! 👍